packages/bitcore-node/lib/routes/wallet.js

```diff
@@ -150,10 +175,10 @@ ListTransactionsStream.prototype._transform = function(transaction, enc, done) {
   done();
 };
 
-router.get('/:walletId/transactions', async (req, res) => {
-  let {walletId, chain, network} = req.params; 
+router.get('/:pubKey/transactions', async (req, res) => {
```

Comment:
Why doesn't this have authenticate, but the utxos does?

Reply:
it should, all wallet methods should. Create wallet is the only one that has diminished value, but even that could if the incoming pubkey is used for verification